mk/check.mk

```diff
@@ -89,4 +89,4 @@ check: format/common lint ## Dev: Run code checks (go fmt, go vet, ...)
 
 .PHONY: update-vulnerable-dependencies
 update-vulnerable-dependencies:
-	@$(KUMA_DIR)/tools/ci/update-vulnerable-dependencies.sh
+	@$(KUMA_DIR)/tools/ci/update-vulnerable-dependencies/update-vulnerable-dependencies.sh
```

Comment:
Can we avoid a shell script that calls a go tool? I think we can as all of the go toolset is available in the SDK.
Also we could also wrap the osv-scanner in there.

Maybe it's even worth moving this to: https://github.com/kumahq/ci-tools

Reply:
cf: https://pkg.go.dev/golang.org/x/mod/modfile

Reply:
I think the whole tool would a lot simpler too.